build.gradle

```diff
@@ -1,45 +1,52 @@
-group 'com.github.simonpercic.responseecho'
-version VERSION_NAME
-
 buildscript {
+    ext.kotlin_version = '1.3.61'
     repositories {
         mavenCentral()
+        maven {
+            url  "https://dl.bintray.com/kaustav07/com.github.kaustav07"
+        }
     }
-
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.6.RELEASE")
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.2.2.RELEASE")
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 apply plugin: 'java'
+apply plugin: 'kotlin'
 apply plugin: 'idea'
-apply plugin: 'spring-boot'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
 
-mainClassName = "com.github.simonpercic.responseecho.Application"
+group 'com.github.simonpercic.responseecho'
+version VERSION_NAME
 
-jar {
-    baseName = 'responseecho'
-    version = VERSION_NAME
+bootJar {
+    archiveBaseName = 'responseecho'
+    archiveVersion = VERSION_NAME
 }
 
 repositories {
+    //mavenLocal()
     jcenter()
+    maven {
+        url  "https://dl.bintray.com/kaustav07/com.github.kaustav07"
+    }
     mavenCentral()
 }
 
 sourceCompatibility = 1.8
-targetCompatibility = 1.8
 
 ext.retrofitVersion = '2.1.0'
 
 dependencies {
-    compile 'com.github.simonpercic:oklog-shared:2.2.0'
+    implementation 'com.github.kaustav07:oklog-shared:2.8.0'
```

Comment:
It seems like you have made some changed to OkLog-Shared as well. Please open a pull request in the other repository, [here](https://github.com/simonpercic/OkLog/tree/master/shared).